packages/frontend/src/pages/campaigns/index.tsx

```diff
@@ -186,8 +177,13 @@ export const Campaigns = () => {
                                                   );
                                               })}
                                     </div>
-                                    <Pagination />
-                                </>
+
+                                    <Pagination
```

Comment:
The `Pagination` shouldn't be visible if the KPI tokens are still loading:

```tsx
{!loading && data.length > 0 && <Pagination />}
```

Reply:
True. Thanks!

Reply:
fixed: cf4ac14e2f9252827bc786c3e9b4f006b3c884cb